pkg/arvo/sys/arvo.hoon

```diff
@@ -1,85 +1,177 @@
+::  +hypo: type associated
+::
++*  hypo  [a]  (pair type a)
+::  +hobo: kiss wrapper
+::
++*  hobo  [a]
+  $?  $%  [%soft p=*]
+      ==
+      a
+  ==
+::  +mark: content type
+::
++$  mark  @tas
+::  +mash:  producing mass
+::
+++  mash  |=(* (mass +<))
+::  +mass: memory usage
+::
++$  mass  $~  [%$ [%& ~]]
+          (pair cord (each noun (list mash)))
+::  +mill: vase+metavase
+::
++$  mill  (each vase milt)
+::  +milt: metavase
+::
++$  milt  [p=* q=*]
+::  +monk: general identity
+::
++$  monk  (each ship [p=@tas q=@ta])
+::  +muse: source move
+::
++$  muse  [p=@tas q=duct r=arvo s=@ud]
+::  +move: arvo move
+::
++$  move  [p=duct q=arvo]
+::  +ovum: typeless ovum
+::
++$  ovum  [p=wire q=curd]
+::  +pane: kernel modules
+::
++$  pane  (list [p=@tas q=vase])
+::  +pone: kernel modules (old)
+::
++$  pone  (list [p=@tas q=vise])
+::  +scry-sample: scry sample
+::
 +$  scry-sample
   [fur=(unit (set monk)) ren=@tas why=shop syd=desk lot=coin tyl=path]
+::  +vane-sample: vane sample
+::
 +$  vane-sample
   [our=ship now=@da eny=@uvJ ski=slyd]
-++  ship  @p                                            ::  network identity
-++  sink  (trel bone ship path)                         ::  subscription
-++  sley  $-  {* (unit (set monk)) term beam}           ::  namespace function
-          (unit (unit cage))                            ::
-++  slyd  $-  {* (unit (set monk)) term beam}           ::  super advanced
-          (unit (unit (cask milt)))                     ::
-++  slyt  $-({* *} (unit (unit)))                       ::  old namespace
+::  +ship: network identity
+::
++$  ship  @p
+::  +sink: subscription
+::
++$  sink  (trel bone ship path)
+::  +sley: namespace function
+::
++$  sley  $-  [* (unit (set monk)) term beam]
+          (unit (unit cage))
+::  +slyd: super advanced
+::
++$  slyd  $-  [* (unit (set monk)) term beam]
+          (unit (unit (cask milt)))
+::  +slyt: old namespace
+::
++$  slyt  $-([* *] (unit (unit)))
+::  +vane: vane
+::
 +$  vane  [=vase =worm]
-++  vile                                                ::  reflexive constants
-          $:  typ/type                                  ::  -:!>(*type)
-              duc/type                                  ::  -:!>(*duct)
-              pah/type                                  ::  -:!>(*path)
-              mev/type                                  ::  -:!>([%meta *vase])
-          ==                                            ::
-++  wind                                                ::  new kernel action
-          |*  {a/mold b/mold}                           ::  forward+reverse
-          $%  {$pass p/path q/a}                        ::  advance
-              {$slip p/a}                               ::  lateral
-              {$give p/b}                               ::  retreat
-          ==                                            ::
-++  wire  path                                          ::  event pretext
+::  +vile: reflexive constants
+::
++$  vile
```

Comment:
As long as there's not tons of items, I find this style excessively verbose compared to:

```
::  +vile: reflexive constants
::
::    typ: -:!>(*type)
::    duc: -:!>(*duct)
::    pah: -:!>(*path)
::    mev: -:!>([%meta *vase])
::
$:  typ=type
    duc=type
    pah=type
    mev=type
==
```

Reply:
Resolving as I've reverted the breathing comments for now.